### PR TITLE
Complete jquery-ajax diagnostic

### PR DESCRIPTION
**Pull request by philschoof**

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -21,13 +21,20 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
...
+curl --include --request GET "http://www.example.com/donuts"
 ```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = $.ajax({
+  method: "GET",
+  url: http://www.example.com/donuts/,
```
Comment (RealWeeks) [inline diagnostic.md:32]: will blow up, it's important that the url be in strings

File: diagnostic.md [hunk 2]
```diff
@@ -36,13 +43,21 @@ Now, we want to get a single donut from the server. Write the `curl` request
...
+
 ```
 
 Write an AJAX request to retrieve a single donut from the server.
 
 ```js
-let getDonut = /* your answer here */;
+let getDonut = $.ajax({
+  method: "GET",
+  url: http://www.example.com/donuts/ + id,
```
Comment (RealWeeks) [inline diagnostic.md:55]: string

File: diagnostic.md [hunk 3]
```diff
@@ -21,13 +21,20 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
...
 ```sh
-# your answer here
+curl --include --request GET "http://www.example.com/donuts"
 ```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = $.ajax({
```
Comment (RealWeeks) [inline diagnostic.md:30]: can't assign ajax to a variable, have to wrap it in a function

File: diagnostic.md [hunk 4]
```diff
@@ -36,13 +43,21 @@ Now, we want to get a single donut from the server. Write the `curl` request
...
-# your answer here
+curl --include --request GET "http://www.example.com/donuts/$ID"
+
 ```
 
 Write an AJAX request to retrieve a single donut from the server.
 
 ```js
-let getDonut = /* your answer here */;
+let getDonut = $.ajax({
```
Comment (RealWeeks) [inline diagnostic.md:53]: same thing about ajax in variables

File: diagnostic.md [hunk 5]
```diff
@@ -51,13 +66,17 @@ Write the `curl` request you'd use to delete a single donut, using whatever ID
...
+curl --include --request DELETE "http://www.example.com/donuts/$ID"
 ```
 
 Write an AJAX request to delete a single donut from the server.
 
 ```js
-let deleteDonut = /* your answer here */;
+let deleteDonut = $.ajax({
+  method "DELETE",
+  url: http://www.example.com/donuts/ + id,
```
Comment (RealWeeks) [inline diagnostic.md:77]: string

File: diagnostic.md [hunk 6]
```diff
@@ -73,13 +92,27 @@ data in JSON format.
...
+      "price": "$0.99"
+    }
+  }'
 ```
 
 Write an AJAX request to create a single donut on the server using JSON.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = $.ajax({
```
Comment (RealWeeks) [inline diagnostic.md:108]: same from earlier comments

File: diagnostic.md [hunk 7]
```diff
@@ -73,13 +92,27 @@ data in JSON format.
...
+  method: 'PATCH',
+  url: 'http://www.example.com/donuts/' + id,
+  processData: false,
```
Comment (RealWeeks) [inline diagnostic.md:111]: need to pass some sort of data to patch.. but this should be a post request

File: diagnostic.md [hunk 8]
```diff
@@ -98,11 +131,25 @@ JSON.
...
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut =   $.ajax({
+    method: 'PATCH',
+    url: 'http://www.example.com/donuts/' + id,
+    processData: false,
```
Comment (RealWeeks) [inline diagnostic.md:149]: no data in here

File: diagnostic.md [hunk 9]
```diff
@@ -98,11 +131,25 @@ JSON.
...
+    "name": "Kruller"
+  }
+}'
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut =   $.ajax({
```
Comment (RealWeeks) [inline diagnostic.md:146]: cant save ajax to a variable

**Comment by RealWeeks** [comment]
5/10.  I left detailed comments.  Keep up the strong effort on this. small changes

**Comment by philschoof** [comment]
Cool - thanks for the comments

**Pull request closed without merging**